src/core/dbus-automount.c

```diff
@@ -35,6 +36,9 @@ static int bus_automount_set_transient_property(
         if (streq(name, "Where"))
                 return bus_set_transient_path(u, name, &a->where, message, flags, error);
 
+        if (streq(name, "Options"))
+                return bus_set_transient_string(u, name, &a->options, message, flags, error);
+
         if (streq(name, "TimeoutIdleUSec"))
                 return bus_set_transient_usec_fix_0(u, name, &a->timeout_idle_usec, message, flags, error);
 
```

Comment:
please also update bus_append_automount_property() in src/shared/bus-unit-util.c (i.e. the client side for systemd-mount